### ISSUE TITLE
Performance in beta with the new Dominos

### ISSUE THREAD

**Issue by boomboo** [tullamods/tullaCC#21]
Seems like the newest version of Dominos causes some performance regression with tullacc.
This is with the previous beta
![oldDominos](https://user-images.githubusercontent.com/175188/90345854-2a5f3000-dfd9-11ea-96c5-9a330db076af.png)

this one is with the new version
![newDominos](https://user-images.githubusercontent.com/175188/90345866-44007780-dfd9-11ea-82e5-fc472cccd6be.png)

**Comment by Tuller** [comment]
Can you comment out the code in https://github.com/tullamods/Dominos/blob/de9f380e031b6daf9ac175d8aff98de6de095ae3/Dominos/bars/actionButtonMixin.lua#L88

And see what happens?

**Comment by boomboo** [comment]
I comented out the functionality, no significant changes or none at all.

**Comment by boomboo** [comment]
I'm not sure why or how but version 4 fixed the issue, I retested with v3 and it was still there but v4 fixes it.

**Comment by Tuller** [comment]
Dominos defers action button creation a bit in beta 4, do you happen to shrink your unused action bars?

**Comment by boomboo** [comment]
Yes, I usually shrink (1x1) and hide the ones I don't use.

**Comment by Tuller** [comment]
Yeah that’ll cause those buttons to never be created by dominos (provided they didn’t exist already) and is likely the cause of your CPU usage reduction

**Comment by boomboo** [comment]
Oh so you mean the reduction is incidental in that whatever caused the increase is still there however with v4 there are less buttons to work on?

**Comment by Tuller** [comment]
I would suspect that the per button CPU usage is about the same between the versions. The only difference was Dominos creating/hooking all 120 action buttons even if you'd shrunk your bars
